packages/table/examples/cellLoadingExample.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as React from "react";
+
+import { RadioGroup } from "@blueprintjs/core";
+import BaseExample, { handleStringChange } from "@blueprintjs/core/examples/common/baseExample";
+
+import { Cell, Column, Table } from "../src";
+
+// tslint:disable-next-line:no-var-requires
+const bigSpaceRocks = require("./pha.json") as IBigSpaceRock[];
```

Comment:
1. awkward to use interface before it's defined.
2. can we give the file a better name? what is "pha?"